### ISSUE TITLE
QA Ph2_Accessibility: Home - Missing alternative text and empty links 

### ISSUE THREAD

**Issue by Remiqa** [GoogleChrome/ioweb2016#381]
There are a number of errors on the home page when using the WAVE extension for Chrome, set to filter WCAG 2.0 AA.  
-1 case of missing alternative text 
-2 cases of empty links 

Please note: I'm unable to get the code feature on WAVE to expand to give a detailed screenshot of where the issues are found. 

Steps: 
1. Go to https://phase2-dot-io-webapp-staging.appspot.com/io2016/
2. Enable the WAVE extension
3. View Style errors with the filter set to WCAG 2.0 AA 
4. Note three errors 

Browser: 
Chrome 48 Mac OS X 10.11 

![gio_381](https://cloud.githubusercontent.com/assets/7585604/13580038/3da4188a-e455-11e5-9af9-179695091af3.png)
